### PR TITLE
Hand Poses

### PR DESCRIPTION
**Pull request by artumino**
Adds hand poses when interacting with items and tools

### CODE REVIEW

File: SteamVR/SteamVR/Input/SteamVR_Skeleton_Poser.cs [hunk 1]
```diff
@@ -436,26 +437,26 @@ public enum BlenderTypes
...
+        //}
+        //
+        //public Quaternion GetTargetHandRotation(SteamVR_Behaviour_Skeleton hand, Transform origin)
+        //{
+        //    Quaternion oldOrigin = origin.rotation;
+        //    origin.rotation = hand.transform.rotation * GetBlendedPose(hand).rotation;
+        //    Quaternion offsetRot = Quaternion.Inverse(origin.rotation) * hand.transform.rotation;
+        //    origin.rotation = oldOrigin;
+        //    return origin.rotation * offsetRot;
+        //}
```
Comment (Raicuparta) [inline SteamVR/SteamVR/Input/SteamVR_Skeleton_Poser.cs:459]: so what's this about?

**Review by Raicuparta — APPROVED** [review]

**Comment by Raicuparta** [comment]
Tested, works perfect. But what's up with the commented out code?

**Comment by artumino** [comment]
The change to the file is a quick fix of a bug with mirrored poses with the SteamVR plugin.
The section commented out are methods that Valve marked as not working, and since after the code changes they were giving compilation errors I didn't bother to change them and commented them instead. Those methods are not used anywhere.

**Comment by Raicuparta** [comment]
@artumino got it. Merge away then, lemme know when you want a release

**Pull request merged**